tensorflow/c/eager/c_api.h

```diff
@@ -81,6 +81,18 @@ TF_CAPI_EXPORT extern void TFE_ContextOptionsSetAsync(TFE_ContextOptions*,
 TF_CAPI_EXPORT extern void TFE_ContextOptionsSetDevicePlacementPolicy(
     TFE_ContextOptions*, TFE_ContextDevicePlacementPolicy);
 
+// Infer type and number attributes from the operation inputs, disabled by
+// default.
+//
+// Enabling it replicates the inference behaviour present in graph construction.
+// It is assumed that inputs are added to the operaton in the same order as they
+// are declared in its definition.
+//
+// It is also assumed that inferable attributes are not set explicitly when
```

Comment:
Can we either remove this assumption or stop setting inferrable attributes in the python op gen?

Reply:
The first assumption cannot be avoided since inputs are added without providing a name (see `TFE_OpAddInput`) but since it always seemed to be the case (with or without inference), then there is no reason to mention it here, I guess.

For the second, as @iganichev mentioned below, it is not a very important assumption since setting a attribute twice will simply override the previous value, so I'll remove that comment as well.